performance/generate.js

```diff
@@ -0,0 +1,29 @@
+const { JSDOM } = require('jsdom');
```

Comment:
enige bedenking die ik me nog maak bij deze opzet. stel dat er in dit script iets fout zit, of dat de component er iets anders uitziet waardoor dit script een pagina genereert die eigenlijk niks rendert, wat gebeurt er dan met uw testen?

Reply:
Die zullen slagen maar uit het rapport zal duidelijk zijn dat er niets getest is.

Reply:
meaning dat er iemand manueel naar het rapport moet kijken?

Reply:
Of we moeten de testdata testen ...

Reply:
ja ik zou toch graag iets hebben dat ook automatisch faalt als er eigenlijk geen deftige pagina getest is ...

Reply:
Extra controle toegevoegd.